src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -476,28 +505,71 @@ class SignUpModalController {
   }
 
   injectErrorMessages (errors = this.signUpErrors) {
```

Comment:
I'm unsure what is broken, but the errors are no longer showing next to the fields and I'm unable to see the errors on step 3

Reply:
I've made sure I downloaded your latest version.

Reply:
The errors are showing up now. Unsure why they weren't before.
<img width="404" alt="Screenshot 2025-03-07 at 3 50 00 PM" src="https://github.com/user-attachments/assets/95af6f29-398b-471b-a17a-5da64ea94e6f" />
<img width="411" alt="Screenshot 2025-03-07 at 3 50 08 PM" src="https://github.com/user-attachments/assets/f6019985-7dc9-47f7-994a-f87c69e4725f" />